Modules/_sqlite/module.c

```diff
@@ -346,6 +346,15 @@ static struct PyModuleDef _sqlite3module = {
         NULL
 };
 
+#define ADD_TYPE(module, type)                 \
+do {                                           \
+    Py_INCREF(&type);                          \
```

Comment:
```suggestion
```

Reply:
Argh, that one snuck back in the revert (commit d96c5e40a9286eeaeddedbdd1d6276fb0eb1a142). Thanks!